src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random randomNumber = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
```suggestion
public class Lottery {
    Random random = new Random();
    ColorSupplier colorSupplier = new ColorSupplier();
    
    public Ball getRandomBall() {
       
```
This are class-fields